software/controller/platformio.ini

```diff
@@ -131,4 +130,4 @@ build_flags =
   ${env.build_flags} -g --coverage -lgcov
 extra_scripts =
   platformio/build_config/platformio_sanitizers.py
-build_src_filter = +<src/>
+build_src_filter = +<release/>
```

Comment:
but never `debug`?

Reply:
whoops, that should not be here, it is a leftover from an investigation to the integration test problem...